### ISSUE TITLE
Error: Cannot find module 'pull-stream/pull'

### ISSUE THREAD

**Issue by elanperach** [pull-stream/stream-to-pull-stream#8]
```
Error: Cannot find module 'pull-stream/pull'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at Object.<anonymous> (/home/elan/Programming/Apps/patchwork/node_modules/stream-to-pull-stream/index.js:1:74)
    at Module._compile (module.js:413:34)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Module.require (module.js:367:17)
```

Fresh install of patchwork and I get this error on "npm start"

**Comment by pfraze** [comment]
@dominictarr I think you intended to update to pull-stream 3.0.0, but didnt update package.json.

**Comment by pfraze** [comment]
Actually I'm not sure what was supposed to happen, but yeah it's broken

**Comment by dominictarr** [comment]
oops, fixed. somehow pull-stream@2 had made it into both deps and devdeps, which confused npm, i think.

**Comment by dominictarr** [comment]
fixed in 1.6.8

**Comment by pfraze** [comment]
👍
